CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.0.0 - 2019-03-12
+This is a major feature release.
+
+Feature:
+  * (RE-12093) Remove old rpm and deb artifacts previously used to build packages.
+  * Remove legacy build action that used the removed artifacts. Packages can only
+  be built using FPM.
+
```

Comment:
@underscorgan updated

Reply:
can you switch legacy build to `legacy-build` for extra clarity?